components/script/dom/htmlbodyelement.rs

```diff
@@ -125,6 +125,14 @@ impl VirtualMethods for HTMLBodyElement {
         Some(self.upcast::<HTMLElement>() as &VirtualMethods)
     }
 
+    fn attribute_affects_presentational_hints(&self, attr: &Attr) -> bool {
+        if attr.local_name() == &local_name!("bgcolor") {
```

Comment:
You could check also the attribute namespace so it's more fine-grained, but it's not a big deal.

Reply:
I'm happy to add it, but my attempts at doing something like

```rust
        if attr.namespace() == &ns!(body) &&
            attr.local_name() == &local_name!("bgcolor") {
            return true;
        }
```

didn't work out. How do I check the namespace for `body`?

Reply:
You want `ns!(html)`, I think